README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://github.com/mikeizbicki/pg_rollup/workflows/tests/badge.svg)](https://github.com/mikeizbicki/pg_rollup/actions)
 
-## TODO
+## TODO~
```

Comment:
You shouldn't make this change.  In general, we don't want any changes to the repo that aren't directly related to the fix that the pull request is about.